templates/configmap.yaml

```diff
@@ -92,61 +92,17 @@ data:
     tier = airflow
     component = worker
     release = {{ .Release.Name }}
-    platform = {{ .Values.platform.release }}
-    workspace = {{ .Values.platform.workspace }}
-{{- end }}
-
-    [astronomer]
-{{- if .Values.webserver.jwtSigningCertificateSecretName }}
-    jwt_signing_cert = {{ template "airflow_webserver_jwt_cert_path" . }}
-    jwt_audience = {{ printf "%s/%s" ( include "deployments_subdomain" . ) .Release.Name }}
+    {{- range $key, $value := .Values.labels }}
```

Comment:
did you have a chance to test it with in real platform/cluster? does it practically work?